app/models/chargeback_rate_detail.rb

```diff
@@ -36,9 +67,12 @@ def find_rate(value)
 
   def cost(value)
     return 0.0 unless self.enabled?
-    value = 1 if group == 'fixed'
+
+    value = 1.0 if fixed?
+
     (fixed_rate, variable_rate) = find_rate(value)
-    hourly(fixed_rate) + hourly(variable_rate) * value
+
+    hourly(fixed_rate) +  hourly(variable_rate) * value
```

Comment:
After working through this with @lpichler and confirming with @sergio-ocon, we came up with a slightly different change - 
```{diff}
diff --git a/app/models/chargeback_rate_detail.rb b/app/models/chargeback_rate_detail.rb
index 651249a..0ebbd48 100644
--- a/app/models/chargeback_rate_detail.rb
+++ b/app/models/chargeback_rate_detail.rb
@@ -72,10 +72,10 @@ class ChargebackRateDetail < ApplicationRecord
 
     (fixed_rate, variable_rate) = find_rate(value)
 
-    hourly(fixed_rate) +  hourly(variable_rate) * value
+    hourly(fixed_rate, true) + (hourly(variable_rate) * value)
   end
 
-  def hourly(rate)
+  def hourly(rate, fixed=nil)
     hourly_rate = case per_time
                   when "hourly"  then rate
                   when "daily"   then rate / 24
@@ -85,7 +85,7 @@ class ChargebackRateDetail < ApplicationRecord
                   else raise "rate time unit of '#{per_time}' not supported"
                   end
 
-    rate_adjustment(hourly_rate)
+    fixed ? hourly_rate : rate_adjustment(hourly_rate)
   end
 
   # Scale the rate in the unit difine by user to the default unit of the metric
```
Fixed rates whether in a tier or in fixed compute or storage should not involve the value of any metric. The only exception a fixed rate is in a tier, the metric value is used to find the correct tier, but in the calculation of the fixed cost.

See [this screen shot](http://screencast.com/t/63i5nHbz7UWm) for reference.